pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1143,7 +1136,7 @@ cdef class _Timedelta(timedelta):
         return fmt.format(**comp_dict)
 
     def __repr__(self) -> str:
-        return "Timedelta('{val}')".format(val=self._repr_base(format='long'))
+        return f"Timedelta('{self._repr_base(format='long')}')"
```

Comment:
can you do the repr_base call outside of fspace

Reply:
Just do
```python
repr_based = self._repr_base(format='long')
return f"Timedelta('{repr_based}')"
```
?